scout/adapter/mongo/variant.py

```diff
@@ -377,11 +377,8 @@ def check_causatives(self, case_obj=None, institute_obj=None, limit_genes=None):
             return []
 
         if case_obj:
-            # exclude variants that are marked causative in "case_obj"
-            case_causative_ids = set(case_obj.get('causatives', []))
-            institute_causative_variant_ids = list(
-                set(institute_causative_variant_ids).difference(case_causative_ids)
-            )
+            institute_causative_variant_ids = [id for id in institute_causative_variant_ids
+                if id not in case_obj.get('causatives',[])]
```

Comment:
👍

Reply:
I'm not done yet with this, Testing some stuff first!